README.md

```diff
@@ -70,10 +70,16 @@ module "kms_all_inclusive" {
   key_protect_instance_name = "my-key-protect-instance"
   resource_group_id         = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
   region                    = "us-south"
-  key_map = {
-    "example-key-ring-1" = ["example-key-1", "example-key-2"]
-    "example-key-ring-2" = ["example-key-3", "example-key-4"]
-  }
+  keys = [
+    {
+      key_ring_name = "example-key-ring-1"
+      keys = [ { key_name = "example-key-1" }, { key_name = "example-key-2" } ]
+    },
+    {
+      key_ring_name = "example-key-ring-2"
+      keys = [ { key_name = "example-key-3" }, { key_name = "example-key-4" } ]
+    }
+  ]
```

Comment:
perhaps format this like pretty json

Reply:
I would also add the other optional options in the keys to show what users can set

Reply:
@MatthewLemmond maybe you missed second comment here?